core/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -355,7 +355,8 @@ class Router(formatOps: FormatOps) {
           case _ => NoSplit
         }
         Seq(
-          Split(modification, 0)
+          Split(modification, 0),
+          Split(if (leftOwner.parent.exists(_.is[Mod])) Space else NoSplit, 0)
```

Comment:
Understood about the PR. Remove it and replace it with what!? Apologies for bugging you, soon will understand the code base :)